reactor-core/src/main/java/reactor/core/publisher/FluxSink.java

```diff
@@ -31,12 +31,10 @@
  * <p>
  * @param <T> the value type
  */
-public interface FluxSink<T> {
+public interface FluxSink<T> extends SequenceSink<T> {
 
-	/**
-     * @see Subscriber#onComplete()
-     */
-    void complete();
+	@Override
```

Comment:
shouldn't it inherit the javadocs?

Reply:
the javadoc is automatically copied, inheritDoc is only if you want to supplement it with additional content while including the `super` content